src/wp-includes/css/custom-properties.css

```diff
@@ -204,4 +214,78 @@ body {
 	--wp-admin--admin-menu--notification--color: var(--wp-admin--theme--notification--contrast);
 	--wp-admin--admin-menu--notification--background--current: var(--wp-admin--admin-menu--notification--background);
 	--wp-admin--admin-menu--notification--color--current: var(--wp-admin--admin-menu--notification--color);
+
+	/* black grayscale rgba # = 0, 0, 0, # */
+	--wp-admin--themes--black-0: rgba(0, 0, 0, 0);
+	--wp-admin--themes--black-05: rgba(0, 0, 0, 0.05);
+	--wp-admin--themes--black-2: rgba(0, 0, 0, 0.2);
+	--wp-admin--themes--black-5: rgba(0, 0, 0, 0.5);
+	--wp-admin--themes--black-6: rgba(0, 0, 0, 0.6);
+	--wp-admin--themes--black-7: rgba(0, 0, 0, 0.7);
+
+	/* white grayscale rgba */
+	--wp-admin--themes--white-65: rgba(255, 255, 255, 0.65);
+	--wp-admin--themes--white-7: rgba(246, 247, 247, 0.7);
+	--wp-admin--themes--white-9: rgba(240, 240, 241, 0.9);
+
+	/* unique rgba colors */
+	--wp-admin--themes--alpha-dark: rgba(44, 51, 56, 0.7);
+	--wp-admin--themes--alpha-gray: rgba(140, 143, 148, 0.1);
+	--wp-admin--themes--blue: rgba(34, 113, 177, 0.8);
+	--wp-admin--themes--light-blue: rgba(79, 148, 212, 0.8);
+
+	/* Misc Box Shadows */
+	--wp-admin--themes--theme--box-shadow: 0 1px 1px -1px var(--wp-admin--surface--box-shadow);
+	--wp-admin--themes--theme-overlay--box-shadow: 0 0 0 1px var(--wp-admin--themes--modal--box-shadow);
+	--wp-admin--themes--theme-name--box-shadow: inset 0 1px 0 var(--wp-admin--surface--box-shadow);
+	--wp-admin--themes--theme-wrap--box-shadow: 0 1px 20px 5px var(--wp-admin--surface--box-shadow);
+	--wp-admin--themes--overlay--box-shadow: 0 0 0 1px var(--wp-admin--themes--icon--shadow--hover),
+											 0 0 2px 1px var(--wp-admin--themes--surface--box-shadow);
+	--wp-admin--themes--overlay-footer--box-shadow: -20px 0 10px -5px var(--wp-admin--page--background);
+	--wp-admin--themes--input-checked--box-shadow: inset 0 2px 5px -3px var(--wp-admin--themes--card--box-shadow);
+	--wp-admin--themes--input-radio-checked--box-shadow: inset 0 2px 5px -3px var(--wp-admin--themes--card--box-shadow),
+														 0 0 3px var(--wp-admin--radio--box-shadow);
+
+	/* Page Level Elements */
+	--wp-admin--themes--surface--box-shadow: var(--wp-admin--themes--light-blue);
+
+	/* misc individual elements */
+	--wp-admin--themes--control--background: #50575e;
+
+	/* icon */
+	--wp-admin--themes--icon--background: #dcdcde;
+	--wp-admin--themes--icon--background--hover: var(--wp-admin--theme--primary);
+	--wp-admin--themes--icon--color: #8c8f94;
+	--wp-admin--themes--icon--color--hover: var(--wp-admin--surface--background);
+	--wp-admin--themes--icon--color--hover-contrast: var(--wp-admin--theme--black);
+	--wp-admin--themes--icon--shadow--hover: #4f94d4;
+	--wp-admin--themes--icon--background--disabled: var(--wp-admin--surface--border);
+
+	/* Theme Card */
```

Comment:
Can these card properties be moved to a more generic `card` set? Or maybe it could use the surface properties?

Reply:
Which would you prefer as a best case? More generic card set or just inherit the surface properties directly? I tried to break up the file into components as much as possible to make it slightly easier to dig through, so I'd lean more towards generic card set personally.